lib/facter/self_service.rb

```diff
@@ -130,4 +130,8 @@
     # Is Hiera 5 in use?
     { S0033: hiera_config_file.dig('version') == 5 }
   end
+  chunk(:S0040) do
+    # Is puppet_metrics_collector::system configured and collecting data
+    { S0040: File.exist?('/etc/systemd/system/puppet_system_processes-metrics.timer') }
```

Comment:
This will check if the file exists, but not if the timer is actually running.  We may be able to use something like [this shared code](https://github.com/puppetlabs/puppetlabs-self_service/blob/b421b77fdd1386b404a169b6b98beea0b3fb0ab7/lib/shared/puppet_self_service.rb#L9-L15) to check that the timer is actually in a running state.  Using `Puppet::Type` should let us inspect the actual state of the service.

Reply:
I have been unable to get that function, or the Puppet::Type function to return the status of *.timer objects I'm not sure whats wrong